containers/containers_shared.mak

```diff
@@ -6,7 +6,7 @@ SANITIZED_DOCKER_REPO = $(subst /,_,$(DOCKER_REPO))
 DOTFILE_IMAGE = $(subst /,_,$(IMAGE))-$(VERSION)
 
 container: container-name
-	docker buildx build --platform linux/amd64 -o type=docker -t $(DOCKER_REPO):$(VERSION) $(DOCKER_ARGS) --label "build-info=$(DOCKER_REPO):$(VERSION) commit=$(shell git describe --tags --always)" .
+	docker buildx build -o type=docker -t $(DOCKER_REPO):$(VERSION) $(DOCKER_ARGS) --label "build-info=$(DOCKER_REPO):$(VERSION) commit=$(shell git describe --tags --always)" .
```

Comment:
This doesn't work on arm64, gets a syntax error. 
```
$ make container
Makefile:21: warning: overriding recipe for target 'push'
../containers_shared.mak:15: warning: ignoring old recipe for target 'push'
container: drud/ddev-webserver:v1.16.0-alpha8-35-g192ca359
docker buildx build -o type=docker --label com.ddev.buildhost=pi1.thefays.us --target=ddev-webserver  -t drud/ddev-webserver:v1.16.0-alpha8-35-g192ca359  .
unknown shorthand flag: 'o' in -o
See 'docker --help'.
```

I also see
```
Makefile:21: warning: overriding commands for target `push'
../containers_shared.mak:15: warning: ignoring old commands for target `push'
```
Which probably says to me that we're not using "push" from containers_shared.mak and it should be removed?

Reply:
Moved this to https://github.com/drud/ddev/issues/2588, so marking this resolved here.